HomeWork/src/main/java/exception/ArrayValueCalculator.java

```diff
@@ -0,0 +1,55 @@
+package exception;
+
+import string.StringLevelOne;
+
+import java.util.Arrays;
+import java.util.zip.DataFormatException;
+
+public class ArrayValueCalculator {
+
+    int results = 0;
+    String[][] str = new String[4][4];
+
+    public void arrayInitialization(String[][] strings1) {
```

Comment:
`strings1` is a bad naming for a parameter. Would be better to name it `values` or `matrix`